ci/test-install/storage_install_test.cc

```diff
@@ -47,8 +47,10 @@ int main(int argc, char* argv[]) try {
   while (std::getline(stream, line, '\n')) {
     ++count;
   }
-  client.DeleteObject(bucket_name, object_name,
-                      gcs::Generation(meta.generation()));
+  client
+      .DeleteObject(bucket_name, object_name,
+                    gcs::Generation(meta.generation()))
+      .value();
```

Comment:
This is a little confusing as the value is not used. Is this meant to throw if the DeleteObject request isn't successful?

Reply:
`DeleteObject` returns a `StatusOr<void>`, `.value()` does throw if there is an error, but there is no way to "use" the `void` value.  The rest of the code uses exceptions to signal errors, If we move to use explicit error checking that should be in a separate PR.

Reply:
Sg, thanks for clarifying.